app/addons/documents/index-results/actions.js

```diff
@@ -50,20 +50,23 @@ export default {
 
     if (!options.collection.fetch) { return; }
 
-    return options.collection.fetch({reset: true}).then(function () {
+    return options.collection.fetch({reset: true}).then(() => {
       this.resultsListReset();
       this.sendMessageNewResultList(options);
 
-    }.bind(this), function (collection, _xhr) {
+    }, function (collection, _xhr) {
       //Make this more robust as sometimes the colection is passed through here.
       var xhr = collection.responseText ? collection : _xhr;
       var errorMsg = 'Bad Request';
 
       try {
-        var responseText = JSON.parse(xhr.responseText);
+        const responseText = JSON.parse(xhr.responseText);
         if (responseText.reason) {
           errorMsg = responseText.reason;
         }
+        if (responseText.error && responseText.error === 'not_found') {
+          return;
```

Comment:
Yes, that is intentional. We use the check in the sidebar action to validate if the database exists or not. If this shows a message here we will get two notifications that say the same thing.